src/app/components/match/match.component.html

```diff
@@ -1,115 +1,178 @@
         <h2 style="display: inline;">
           תיאום התנדבות
         </h2>
-
-        <div class="input-group mb-1">
-        <h4 class="secondary-header mt-2 mb-0">
-            בחירת בקשת התנדבות
-        </h4>
-        <table class="table rtl table-sm table-hover" dir="rtl">
-          <thead class="thead">
-          <tr>
-            <th scope="col">שם</th>
-            <th scope="col">מספר מתנדבים</th>
-            <th scope="col">איש קשר</th>
-            <th scope="col">ימות השבוע</th>
-            <th scope="col">מיקום</th>
-            <th scope="col">מספר מופעים</th>
-            <th scope="col">איזור</th>
-          </tr>
-          </thead>
-          <tbody>
-          <tr *ngFor="let request of volunteeringRequests"
-              [class.selected]="request === selectedRequest"
-              (click)="onSelectRequest(request)">
-            <td>{{request.title}}</td>
-            <td>{{request.numberOfVolunteers}}</td>
-            <td>{{request.contact | styleContact}}</td>
-            <td>{{request.availableWeekdays | weekdayNames | join: ', '}}</td>
-            <td>{{request.location}}</td>
-            <td>{{request.numberOfOccurrences}}</td>
-            <td>{{request.region}}</td>
-          </tr>
-          </tbody>
-        </table>
       </div>
-
-        <div class="input-group mb-1">
-          <h4 class="secondary-header mt-2 mb-0">
-            בחירת הצעת התנדבות
-          </h4>
-          <table class="table rtl table-sm table-hover" dir="rtl">
-            <thead class="thead">
-            <tr>
-              <th scope="col">שם</th>
-              <th scope="col">מספר מתנדבים</th>
-              <th scope="col">איש קשר</th>
-              <th scope="col">ימות השבוע</th>
-              <th scope="col">איזורים</th>
-            </tr>
+    </div>
+    <div class="row">
+        <div class="col-5">
+            <h4 class="secondary-header mt-2 mb-0">
+                בחירת בקשת התנדבות
+            </h4>
+        </div>
+        <div class="col-2"></div>
+        <div class="col-5">
+            <h4 class="secondary-header mt-2 mb-0">
+                בחירת הצעת התנדבות
+            </h4>
+        </div>
+    </div>
+    <div class="row mb-1">
+        <div class="col-5">
+            <ul id="accordion" class="list-group p-0 mx-0">
+                <li *ngFor="let request of volunteeringRequests; index as i" class="list-group-item align-items-start p-0 m-0">
+                    <div class="card p-0 m-0">
+                        <div (click)="onSelectRequest(request)" [ngClass]="{'is-selected': selectedRequest===request}"
+                            class="row card-header py-0 mx-0" aria-expanded="true" [attr.aria-controls]="'#collapse'+i" [attr.id]="'headingO'+i">
+                           <div class="col-1 d-flex align-items-center justify-content-center">
+                                <i class="fas fa-plus-circle fa-2x pl-1" data-toggle="collapse" [attr.data-target]="'#collapse'+i"></i>
```

Comment:
I think you should change the icon because plus sign has nothing to do here.
Maybe we can remove it at all.